packages/swap/src/utils/rpc.ts

```diff
@@ -0,0 +1,47 @@
+import { Asset, ChainflipNetwork, assetChains } from '@/shared/enums';
+import {
+  getIngressEgressEnvironment,
+  getSwappingEnvironment,
+  type ChainAssetMap,
+  getPoolsEnvironment,
+} from '@/shared/rpc';
+
+const readAssetValue = (minimums: ChainAssetMap<bigint>, asset: Asset) => {
+  const chain = assetChains[asset];
+  const chainMinimums = minimums[chain];
+  return chainMinimums[asset as keyof typeof chainMinimums];
+};
+
+export const getMinimumDepositAmount = async (
+  network: ChainflipNetwork,
+  asset: Asset,
+): Promise<bigint> => {
+  const ingressEgressEnv = await getIngressEgressEnvironment(network);
+  return readAssetValue(ingressEgressEnv.minimumDepositAmounts, asset);
+};
+
+export const getMinimumSwapAmount = async (
+  network: ChainflipNetwork,
+  asset: Asset,
+): Promise<bigint> => {
+  const swapEnv = await getSwappingEnvironment(network);
+  return readAssetValue(swapEnv.minimumSwapAmounts, asset);
+};
+
+export const getPoolsNetworkFeeHundredthPips = async (
```

Comment:
not sure about the name - it looks like it is returning pool infos, not the network fee (which should be the same for all pools)?

Reply:
also i dont think its used atm right?